graphmodel/condition.json

```diff
@@ -35,56 +35,14 @@
             },
             "https://dartnet.info/graphModel/Condition/source": {
                 "@type": ["Object","Code"],
-                "attributes": {
-                    "https://dartnet.info/graphModel/Condition/source/code": {
-                        "@type": "Value",
-                        "attributeName": "code",
-                        "ingestAs": "property"
-                    },
-                    "https://dartnet.info/graphModel/Condition/source/system": {
-                        "@type": "Value",
-                        "attributeName": "system",
-                        "ingestAs": "property"
-                    },
-                    "https://dartnet.info/graphModel/Condition/source/text": {
-                        "@type": "Value",
-                        "attributeName": "text",
-                        "ingestAs": "property"
-                    }
-                }
+                "include":"source-include.schema.json"
             },
             "https://dartnet.info/graphModel/Condition/omop": {
                 "@type": "Array",
                 "arrayElements": {
                     "@id": "https://dartnet.info/graphmodel/Condition/omop/items",
                     "@type": ["Object","Code","ConditionCode","OMOPConcept"],
-                    "attributes": {
-                        "https://dartnet.info/graphModel/Condition/omop/id": {
-                            "@type": "Value",
-                            "attributeName": "id",
-                            "ingestAs": "property"
-                        },
-                        "https://dartnet.info/graphModel/Condition/omop/name": {
-                            "@type": "Value",
-                            "attributeName": "name",
-                            "ingestAs": "property"
-                        },
-                        "https://dartnet.info/graphModel/Condition/omop/domainId": {
-                            "@type": "Value",
-                            "attributeName": "domainId",
-                            "ingestAs": "property"
-                        },
-                        "https://dartnet.info/graphModel/Condition/omop/vocabularyId": {
-                            "@type": "Value",
-                            "attributeName": "vocabularyId",
-                            "ingestAs": "property"
-                        },
-                        "https://dartnet.info/graphModel/Condition/omop/classId": {
-                            "@type": "Value",
-                            "attributeName": "classId",
-                            "ingestAs": "property"
-                        }
-                    }
+                    "include":"omop-include.schema.json"
```

Comment:
Make OMOP references a Reference

"@type": ["Reference", "Code", "ConditionCode"],

in the OMOP schema:

@type: Object, "OMOPConcept"